client/js/stores/day-store.js

```diff
@@ -48,7 +54,10 @@ var DayStore = assign({}, EventEmitter.prototype, {
 Dispatcher.register(function(payload){
   switch (payload.actionType){
     case AppConstants.ServerActionTypes.DAY_DATA_RECIEVED:
-      _dayData = payload.dayData;
+      _dayData.gigs = _.indexBy(payload.dayData, function(gig){
+        return gig.id;
+      });
+      _dayData.date = payload.dayData[0] && moment(payload.dayData[0].date, 'YYYY-MM-DD') || '';
```

Comment:
just thinking about it - we could also just append startDate and endDate to the data we receive back in the api-utils